osu.Game.Tournament/Screens/Ladder/Components/LadderEditorSettings.cs

```diff
@@ -62,22 +62,28 @@ private void load()
                 // ensure any ongoing edits are committed out to the *current* selection before changing to a new one.
                 GetContainingInputManager().TriggerFocusContention(null);
 
-                roundDropdown.Current = selection.NewValue?.Round;
-                losersCheckbox.Current = selection.NewValue?.Losers;
-                dateTimeBox.Current = selection.NewValue?.Date;
+                // Required to avoid cyclic failure in BindableWithCurrent (TriggerChange called during the Current_Set process).
```

Comment:
The repro for this is something like:

- Have a match populated with date before round start date (to trigger the `TriggerChange` below)
- Click said match
- Click any other match
- Click back on said match
- Crash